config/Cargo.toml

```diff
@@ -31,11 +31,16 @@ displaydoc = { workspace = true }
 derive_more = { workspace = true }
 cfg-if = { workspace = true }
 once_cell = { workspace = true }
+nonzero_ext = "0.3.0"
+toml = { workspace = true }
+merge = "0.1.0"
```

Comment:
I mentioned this elsewhere this dependency is quite useless. This code should be hand written

Reply:
Replied in other place why `merge` is there at all.